lib/Runtime/Library/JsBuiltInEngineInterfaceExtensionObject.cpp

```diff
@@ -453,5 +460,34 @@ namespace Js
         return obj;
     }
 
+    Var JsBuiltInEngineInterfaceExtensionObject::EntryJsBuiltIn_Internal_ArraySpeciesCreate(RecyclableObject *function, CallInfo callInfo, ...)
```

Comment:
future todo : I think ArraySpeciesCreate should be implemented entirely in javascript. May be in future releases